src/store-enhancer.js

```diff
@@ -174,26 +174,26 @@ export default ({
 
     const dispatch = action => {
       switch (action.type) {
-        case PUSH:
-          history.push(action.payload);
-          return null;
-        case REPLACE:
-          history.replace(action.payload);
-          return null;
-        case GO:
-          history.go(action.payload);
-          return null;
-        case GO_BACK:
-          history.goBack();
-          return null;
-        case GO_FORWARD:
-          history.goForward();
-          return null;
-        default:
-          // We return the result of dispatch here
-          // to retain compatibility with enhancers
-          // that return a promise from dispatch.
-          return store.dispatch(action);
+      case PUSH:
```

Comment:
It's weirdly the rule in `eslint-config-formidable`...we had an override before, maybe we rethink this rule there?